### PR TITLE
Feat/add more playlist info

### PR DESCRIPTION
**Pull request by martinstark**
Add FPS and Codec info

How to test:

`nvm use v18 && npm i && npm start`

- To test bitrate detection: play something in Safari
- To test the playlist parsing: add a console log to index.ts playlists, open any browser

### CODE REVIEW

**Comment by martinstark** [comment]
@bwallberg

File: index.ts [hunk 1]
```diff
@@ -97,13 +97,50 @@ export class SafariBitrateMonitor {
...
+              }
+
+              if (codecs.length >= 2) {
+                // Guess based on some of the most common codecs. There is no
+                // way to be 100% certain which codec belongs to which track
+                // in the HLS standard.
+                const firstIsProbablyVideo = codecs[0].includes("avc");
+                const secondIsProbablyVideo = codecs[1].includes("avc");
+
+                const firstIsProbablyAudio = codecs[0].includes("mp4");
```
Comment (bwallberg) [inline index.ts]: `mp4a`

**Review by bwallberg — APPROVED** [review]
I'd recommend just moving out the guessing to somewhere else, and atleast guess on `mp4a` as that is the audio codec base type you're guessing on.

**Review by bwallberg — APPROVED** [review]
LGTM!

**Pull request merged**